src/main/java/com/robosh/ejournal/controller/AdminController.java

```diff
@@ -1,8 +1,32 @@
 package com.robosh.ejournal.controller;
 
+import com.robosh.ejournal.data.dto.admin.AdminInfoDto;
+import com.robosh.ejournal.data.dto.admin.UpdateAdminDto;
+import com.robosh.ejournal.service.AdminService;
+import io.swagger.annotations.ApiOperation;
+import io.swagger.annotations.ApiParam;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import javax.validation.Valid;
+
 @RestController
+@RequestMapping("/admins")
```

Comment:
Why not "/admin" ?

Reply:
due to rest best practice which we talk about some days ago

Reply:
i read the chat, but is it necessary for admin?

Reply:
if it is a convension, it spreads to all classes

Reply:
ok, but it looks weird

Reply:
I agree, but now it made by the convention.

Reply:
i think that was convention for get methods, was not it ?

Reply:
no, it is for all methods